Pear/Profile/EditProfileViewController.swift

```diff
@@ -105,3 +130,16 @@ extension EditProfileViewController: UIPageViewControllerDataSource {
     }
     
 }
+
+/// Passes the user back to the ProfileViewController so that it can be updated there
+extension EditProfileViewController: didEditProfileDelegate {
+    func updateUser(updatedUser: UserV2) {
+        currentUser = updatedUser
+    }
+}
+
+extension EditProfileViewController: didEditDemographicsDelegate {
+    func updateDemographics(updatedUser: UserV2) {
```

Comment:
same thing here. extensions should always have a blank line at the start and at the end